README.md

```diff
@@ -77,3 +80,35 @@ prompt that will allow you to experiment.
 
 Bug reports and pull requests are welcome on GitHub at
 https://github.com/thoughtbot/gold_miner.
+
+## Register your own app with Slack
+
+To register an app with Slack you just need to give it a name and select a
+workspace to develop it. After that, you will be able to generate Bot and User
+Tokens from your App settings page. A User Token is needed instead of a
+Bot Token because Gold Miner uses Slack's [search.messages] API which is only
+available for User Tokens.
+
+- Go to https://api.slack.com/apps and click on "Create New App"
+  and then click "From scratch"
+- Give it a name, select your workspace, and click "Create App"
+- After that, go to "Add features and functionality > Permissions"
+- Scroll down to "Scopes > User Token Scopes" and give your app the
+  following scopes:
+  - `search:read` to have access to [search.messages]
+  - `users:read` to have access to [users.info]
+- Then scroll back up to "OAuth Tokens for Your Workspace"
+  and click "Install to Workspace"
+- You will be redirected to a page asking you to allow your app to be
+  installed to the workspace and listing the scopes/permissons you've selected
```

Comment:
```suggestion
  installed to the workspace and listing the scopes/permissions you've selected
```

Reply:
Good catch, thanks! I may have forgotten to spell-check this